docs/advanced/bootloader.md

```diff
@@ -6,7 +6,11 @@ This guide explains how to create a bootloader, how to create a main program to
 
 A target requires flash HAL functionality in order to support the bootloader. Please follow [the bootloader porting guide](https://docs.mbed.com/docs/mbed-os-handbook/en/latest/advanced/flash/) for more details.
 
-## Creating the bootloader
+## mbed OS managed bootloader
+
+The tools of mbed OS know how to manage some bootloader projects. The tools can manage bootloader projects where the bootlodoader is placed in ROM before the application and the application starts immediately after the bootloader. If your bootloader does not meet both of these requirements, then pleas read the [unmanaged bootloader section](bootloader.md#unmanaged-bootloader). A managed bootloader project will automatically merge the bootloader image with the application image as part of the application image build process.
```

Comment:
- In the second sentence, "is placed in ROM" is passive voice. Please clarify who or what is doing the placing.
- Please change "pleas" to "please" in the third sentence.
- In the last sentence, please change "will automatically merge" to "automatically merges" for consistent tense.